inc/fields/class-shortcode-ui-fields.php

```diff
@@ -96,13 +96,14 @@ public function action_init() {
 		$this->fields = apply_filters( 'shortcode_ui_fields', $this->fields );
 
 		// set default args for each field.
-		$field_defaults = $this->field_defaults;
-		$this->fields = array_map( function( $args ) use ( $field_defaults ) {
-			return wp_parse_args( $args, $field_defaults );
-		}, $this->fields );
+		$this->fields = array_map( array( $this, 'array_map_args' ) , $this->fields );
```

Comment:
Rather than expose a public method, can we just make this a `foreach` loop instead?